src/integrationtest/python/janusgraph_container.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import docker
+from janusgraph_python.driver.janusgraph_remote_connection_builder import JanusGraphRemoteConnectionBuilder
+import time
+
+
+class JanusGraphContainer(object):
+    def __init__(self):
+        self.BASE_IMAGE = "florianhockmann/janusgraph-testing:0.3.0"
```

Comment:
Should this be using the standard JanusGraph container in https://github.com/janusgraph/janusgraph-docker e.g., `janusgraph/janusgraph`?

Reply:
I actually have been thinking the same thing as well. @FlorianHockmann  which docker image do you use with .NET? Also, if I remember correctly, the official docker images don't have `GraphOfGodsFactory` loaded whereas Florian your image have those loaded right?

If that's the reason we might need to do additional step of loading the Graph then run integration tests.